torchvision/utils.py

```diff
@@ -382,6 +389,130 @@ def draw_keypoints(
     return torch.from_numpy(np.array(img_to_draw)).permute(2, 0, 1).to(dtype=torch.uint8)
 
 
+# Flow visualization code adapted from https://github.com/tomrunia/OpticalFlow_Visualization
+@torch.no_grad()
+def flow_to_image(
+    flow: torch.Tensor,
+    *,
+    clip_flow: Optional[float] = None,
+) -> torch.Tensor:
+
+    """
+    Converts the two-dimensional flow to an RGB Image.
+
+    Args:
+        flow (Tensor): Flow of shape (2, H, W)
+        clip_flow (float, optional): Clip maximum of flow values. Defaults to None.
+
+    Returns:
+        img (Tensor(3, H, W)): Image Tensor where each color corresponds to a given flow direction.
+    """
+
+    if flow.ndim != 3 or flow.size(0) != 2:
+        raise ValueError(f"Input flow should have shape (2, H, W), got {flow.shape}.")
+
+    if clip_flow is not None:
+        flow = torch.clip(flow, 0, clip_flow)
+
+    u = flow[0, :, :]
+    v = flow[1, :, :]
+    rad = torch.sqrt(torch.square(u) + torch.square(v))
+    rad_max = torch.max(rad)
+    epslion = 1e-5
+    u = u / (rad_max + epslion)
+    v = v / (rad_max + epslion)
+    return _flow_to_colors(u, v)
+
+
+@torch.no_grad()
+def _flow_to_colors(
```

Comment:
maybe this would be a more descriptive name?

```suggestion
def _normalized_flow_to_image(
```

Reply:
I guess you mean the other function. But I get it.

Reply:
I guess this is internal function and it might be safe to call it colors, as it can confuse the notion of colors and image. 
So will rename to `normalized_flow_to_colors`